### ISSUE TITLE
Cherry-pick for release v0.2.0-rc.1

### ISSUE THREAD

**Issue by kunmingg** [kubeflow/kubeflow#1024]
Open this issue to track all bugs found / cherry-picks to release branch till create release v0.2.0-rc.1.

**Comment by kunmingg** [comment]
Fix Tf-job version in bootstrapper image:
https://github.com/kubeflow/kubeflow/commit/f3df0c9e0d578941d2cbab61956543e3905796e1

**Comment by kunmingg** [comment]
https://github.com/kubeflow/kubeflow/issues/1041

**Comment by jlewi** [comment]
@kunmingg There are a bunch of tf-job-operator fixes that we should pull in on the next RC. So we should plan on grabbing the latest nightly build of the TFJob operator image and then updating the prototype.

**Comment by jlewi** [comment]
Here are some recent we are waiting on:

Critical Fixes

- [ ] kubeflow/tf-operator#689 - Fix TFJob labels
- [ ] kubeflow/tf-operator#691 - Add CleanupPolicy to v1alpha2 
- [ ] kubeflow/kubeflow#1035 - Update examples package
- [ ] kubeflow/kubeflow#1052 - Fix default image for notebook

Nice to have
kubeflow/kubeflow#1043 - MPI operatorksonnet package

**Comment by kunmingg** [comment]
tf-operator head doesn't seem to work, will pick up fixes in next release candidate. 
https://github.com/kubeflow/tf-operator/tree/v0.2-branch
